### ISSUE TITLE
Manage deposits

### ISSUE THREAD

**Issue by agd1027** [ijm1003/GESPRO_Practica_2_Curso_2024_2025#30]

**Comment by agd1027** [comment]
¿Se está registrando correctamente el historial de depósitos en la base de datos?

**Comment by ijm1003** [comment]
Sí, pero me di cuenta de que no está ordenando por fecha. Estoy corrigiendo eso.

**Comment by agd1027** [comment]
Bien, avísame cuando lo tengas para incluirlo en los reportes.
